### ISSUE TITLE
tsDeclPath is not defined

### ISSUE THREAD

**Issue by kenchan0130** [yortus/require-self#13]
Using `require-self` with `0.2.2` does not work with an error.

The actual error log is as follows:

```
/home/travis/build/kenchan0130/babel-preset-google-apps-script/node_modules/require-self/bin/require-self:23
    throw e;
    ^
ReferenceError: tsDeclPath is not defined
    at Object.<anonymous> (/home/travis/build/kenchan0130/babel-preset-google-apps-script/node_modules/require-self/bin/require-self:20:29)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47
```

You can also see it at https://travis-ci.org/github/kenchan0130/babel-preset-google-apps-script/builds/657483044.

## Version

Node.js: v8, v10, v12
require-self: 0.2.2

**Comment by yortus** [comment]
@kenchan0130 should be fixed in v0.2.3.

**Comment by kenchan0130** [comment]
Thank you for your support! 
I’ll try it.
